design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
 
 ![](./assets/video-infrastructure.webp)
 
-When a streamer goes live they hit the first microservice called ingest.
+When a streamer goes live, they hit the first microservice called ingest.
```

Comment:
"ingest" is capitalized right after this. Since it's an important entity you can probably capitalize it to "Ingest".

Reply:
seconded